channels/us.m3u

```diff
@@ -354,9 +354,9 @@ https://bozztv.com/teleyupp/teleup-axshd/playlist.m3u8
 https://dikcfc9915kp8.cloudfront.net/hls/main.m3u8
 #EXTINF:-1 tvg-id="AyenehTV.us" tvg-name="Ayeneh TV" tvg-country="US" tvg-language="Persian" tvg-logo="https://i.imgur.com/TpnmasZ.png" group-title="General",Ayeneh TV (720p)
 https://livestream.5centscdn.com/cls040318/b0d2763968fd0bdd2dc0d44ba2abf9ce.sdp/index.m3u8
-#EXTINF:-1 tvg-id="KEJRLD1.us" tvg-name="Azteca 40.1 (KEJR-LD)" tvg-country="US" tvg-language="English" tvg-logo="https://media-uspx.mybtv.net/logos/40.1-Azteca.png" group-title="",Azteca 40.1 (KEJR-LD) (720p)
+#EXTINF:-1 tvg-id="KEJRLD.us" tvg-name="Azteca 40 (KEJR-LD)" tvg-country="US" tvg-language="English" tvg-logo="https://media-uspx.mybtv.net/logos/40.1-Azteca.png" group-title="",Azteca 40 (KEJR-LD) (720p)
 https://v-px.mybtv.net/event/41/1280x720.m3u8?start_time=2021-05-16T23%3A45%3A00%2B00%3A00&end_time=2023-05-17T05%3A00%3A00%2B00%3A00&env=prd&uid=33630
-#EXTINF:-1 tvg-id="KEJRLD1.us" tvg-name="Azteca 40.1 (KEJR-LD)" tvg-country="US" tvg-language="English" tvg-logo="https://media-uspx.mybtv.net/logos/40.1-Azteca.png" group-title="",Azteca 40.1 (KEJR-LD) (480i)
+#EXTINF:-1 tvg-id="KEJRLD.us" tvg-name="Azteca 40.1 (KEJR-LD)" tvg-country="US" tvg-language="English" tvg-logo="https://media-uspx.mybtv.net/logos/40.1-Azteca.png" group-title="",Azteca 40.1 (KEJR-LD) (480i)
```

Comment:
Looks like this one should be renamed too

Reply:
Which one? I changed them both. Are you referring to the channel number? 40 vs 40.1? They are both. One is 480i and the other is 720p. So it's correct.